packages/wonder-blocks-modal/src/components/modal-launcher.tsx

```diff
@@ -26,7 +26,7 @@ type Props = {
      * Note: Don't call `closeModal` while rendering! It should be used to
      * respond to user intearction, like `onClick`.
      */
-    modal: ModalElement | React.FC<{closeModal: () => void}>;
+    modal: ModalElement | ((props: {closeModal: () => void}) => ModalElement);
```

Comment:
`ModalElement` is defined as `React.ReactElement | null` which is the same as the return type for things typed as `React.FC<>`.